internal/pipe/nfpm/nfpm.go

```diff
@@ -62,6 +62,9 @@ func (Pipe) Default(ctx *context.Context) error {
 				fpm.Builds = append(fpm.Builds, b.ID)
 			}
 		}
+		if fpm.Release == "" {
+			fpm.Release = ctx.Env["RELEASE"]
```

Comment:
I don't get why this default

no other config fields on goreleaser default to environment variable values